src/puppetlabs/dujour/version_check.clj

```diff
@@ -72,7 +74,9 @@
    update-server-url :- schema/Str]
   (let [product-name (:product-name request-values)
         {:keys [group-id artifact-id]} (get-coords product-name)
-        current-version (version group-id artifact-id)
+        ;current-version (version group-id artifact-id (str (or (:version request-values) "")))
+        current-version (str (or (:version request-values) (version group-id artifact-id "")))
+        _ (log/errorf "current-version = %s" current-version)
```

Comment:
This is fixed [here](https://github.com/puppetlabs/dujour-version-check/pull/16).